src/core/manager.c

```diff
@@ -3132,6 +3132,9 @@ int manager_reload(Manager *m) {
         manager_vacuum_uid_refs(m);
         manager_vacuum_gid_refs(m);
 
+        /* It might be safe to log to the journal now. */
+        manager_recheck_journal(m);
```

Comment:
@poettering I'm confused by this.  I'm thinking about where recheck_journal() should be called, knowing that the initrd systemd passes its state to the "real" one.

Don't we need to call recheck_journal() from manager_startup()?

Why do we want to call it from manager_reload()?  The state of the journald units should be the same before and after deserialization.  Surely manager_reload() doesn't somehow check for unit readiness and yet avoid calling unit_notify()?